scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -363,6 +367,99 @@ class FormatWriter(formatOps: FormatOps) {
       finalResult.result()
     }
   }
+
+  import scala.meta.internal.classifiers.classifier
+
+  @classifier
+  trait CloseDelim
+  object CloseDelim {
+    def unapply(token: Token): Boolean =
+      token.is[RightParen] || token.is[RightBracket] || token.is[RightBrace]
+  }
+
+  private def handleTrailingCommasAndWhitespace(
+      formatToken: FormatToken,
+      state: State,
+      sb: StringBuilder,
+      whitespace: String): Unit = {
+
+    import org.scalafmt.config.TrailingCommas
+
+    val owner = owners(formatToken.right)
+    val hasValidOwner =
+      owner.is[Term.Apply] || owner.is[Term.Tuple] ||
```

Comment:
Can we introduce a classifier named `CommaSeparated` for this group? I think `Ctor.{Primary,Secondary}` are missing for 
```scala
class Foo(
  a: Int,
  b: String,
)
```

Reply:
Nice catch, yes I'll introduce a classifier if you think it's easier to read